### ISSUE TITLE
Merge docker's and system's CA certificates

### ISSUE THREAD

**Issue by h0tbird** [h0tbird/kato#19]
Check this task and [comment](https://github.com/docker/distribution/issues/426#issuecomment-95851710) to understand why this is needed.

**Comment by h0tbird** [comment]
This seems to permanently fix the issue:
```
sudo cp /etc/docker/certs.d/internal-registry-sys.marathon:5000/ca.crt /etc/ssl/certs/cell-1.dub.pem                                     
sudo update-ca-certificates
sudo rm -rf /etc/docker/certs.d/internal-registry-sys.marathon:5000
sudo systemctl restart docker
```

**Comment by h0tbird** [comment]
Ops! That will fix the `push` but break the `login`

**Comment by h0tbird** [comment]
http://serverfault.com/a/782936/359718

**Comment by h0tbird** [comment]
Fixed by ee51132365e47997037babcea7c3779bcb9a2788
